code/src/java/pcgen/cdom/enumeration/SkillsOutputOrder.java

```diff
@@ -33,49 +30,9 @@
  */
 public enum SkillsOutputOrder
 {
-	NAME_ASC {
-		@Override
-		public SkillComparator getComparator(PlayerCharacter pc)
-		{
-			return new SkillComparator(pc, SkillComparator.RESORT_NAME,
-				SkillComparator.RESORT_ASCENDING);
-		}
-	},
-
-	NAME_DSC {
-		@Override
-		public SkillComparator getComparator(PlayerCharacter pc)
-		{
-			return new SkillComparator(pc, SkillComparator.RESORT_NAME,
-				SkillComparator.RESORT_DESCENDING);
-		}
-	},
-
-	TRAINED_ASC {
-		@Override
-		public SkillComparator getComparator(PlayerCharacter pc)
-		{
-			return new SkillComparator(pc, SkillComparator.RESORT_TRAINED,
-				SkillComparator.RESORT_ASCENDING);
-		}
-	},
-
-	TRAINED_DSC {
-		@Override
-		public SkillComparator getComparator(PlayerCharacter pc)
-		{
-			return new SkillComparator(pc, SkillComparator.RESORT_TRAINED,
-				SkillComparator.RESORT_DESCENDING);
-		}
-	},
-
-	MANUAL {
-		@Override
-		public SkillComparator getComparator(PlayerCharacter pc)
-		{
-			return null;
-		}
-	};
-
-	public abstract SkillComparator getComparator(PlayerCharacter pc);
+	NAME_ASC,
+	NAME_DSC,
+	TRAINED_ASC,
+	TRAINED_DSC,
+	MANUAL;
 }
```

Comment:
So this is a huge red flag that we have an enumeration that now has no behavior associated with different output orders.  This should be left alone and flagged as a UI bug I think... or it needs a deep UI code review to understand what is going on elsewhere to do this work